src/Day02.kt

```diff
@@ -18,15 +9,10 @@ fun main() {
 
     // Part 1
     fun part1(input: List<String>): Int {
+        var totalPaper = 0
 
-        val measurements = input
-            .map { it.split('x') }
-            .map { it.map { it.toInt() }}
-
-        var totalPaper   = 0
-
-        for (i in measurements.indices) {
-            val (length, width, height) = Triple(measurements[i][0], measurements[i][1], measurements[i][2])
+        for (dimensions in input) {
```

Comment:
Prefer this form of `for` loop when possible. If you don't need the indices it's cleaner to write it like this.